packages/paste-website/src/pages/components/anchor/index.mdx

```diff
@@ -93,21 +101,17 @@ This is done for security purposes. Even though the target and rel are set by de
 You can use an anchor to navigate the user to another webpage.
 
 <DoDont>
-  <Do>
-    Anchors should only be used to link to another page, app, or another website.
+  <Do title="Do" body="Anchors should only be used to link to another page, app, or another website." center>
```

Comment:
What if you want more than one line of text?

Reply:
Do we want to allow more than one line here?

If there needs to be more descriptive text, should that be a do/dont or handled using the page copy?

Reply:
It can merge for now, it's not a blocker

Reply:
Cool. I'm on board with not being a fan of text based props. When we have a use case for changing, we can revisit.